src/extensions/surveys.ts

```diff
@@ -764,6 +777,10 @@ export const createMultipleQuestionSurvey = (posthog: PostHog, survey: Survey) =
                               ].map((choice) => choice.value)
                     response = selectedChoices
                 }
+                const isQuestionOptional = survey.questions[idx].optional
+                if (isQuestionOptional && response === undefined) {
+                    response = null
```

Comment:
@neilkakkar Thoughts on what "skipped" questions should have a response of if not null?

Reply:
null/undefined makes the most sense 👍 . We don't even have to send a response value here.